### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by loganmcmullen**
*Logan Moved all items into controller, model view or test folders to respect MVC architecture
*Riley changed resource paths to allow the fxml files to be found upon running the program
*Riley refactored repetitive code in the SmartHomeSimulatorController.java file (removed hundreds of lines of unnecessary code)

### CODE REVIEW

File: javadoc/index-files/index-5.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h2 class="title" id="I:G">G</h2>
 </dd>
 <dt><span class="member-name-link"><a href="../com/simulator/model/SimulationParameters.html#getAllUserNames()">getAllUserNames()</a></span> - Method in class com.simulator.model.<a href="../com/simulator/model/SimulationParameters.html" title="class in com.simulator.model">SimulationParameters</a></dt>
 <dd>
-<div class="block">Get all user names in a List<String></div>
+    <div class="block">Get all user names in a List<String></String></div>
```
Comment (bowe99) [inline javadoc/index-files/index-5.html:65]: Why do we need a <String></String> at the end

File: src/com/simulator/controller/SmartHomeSimulatorController.java [hunk 2]
```diff
@@ -0,0 +1,841 @@
...
+            Stage stage = (Stage) finished.getScene().getWindow();
+            stage.close();
+           }
+        catch (Exception e){
+        e.printStackTrace();
+        }
+    }
+   
+}
+
```
Comment (bowe99) [inline src/com/simulator/controller/SmartHomeSimulatorController.java:841]: Can you just give a recap as to why the code dropped from 1671 lines to 841

**Review by bowe99 — APPROVED** [review]
Just a couple of comments before merging

File: src/com/simulator/controller/SmartHomeSimulatorController.java (hunk 2 above)
Comment (loganmcmullen) [reply src/com/simulator/controller/SmartHomeSimulatorController.java:841]: Basically there was hundreds of objects that were initialized one by like windowImages, light images, etc and riley reduced them to arrays to store them. Also massive if statements that were repeated were store into methods using extract method refactoring

File: javadoc/index-files/index-5.html (hunk 1 above)
Comment (loganmcmullen) [reply javadoc/index-files/index-5.html:65]: Probabably a mistake, will remove

**Pull request merged**